cmd/kubelet/app/options/options.go

```diff
@@ -86,7 +86,7 @@ type KubeletFlags struct {
 	config.ContainerRuntimeOptions
 
 	// certDirectory is the directory where the TLS certs are located (by
-	// default /var/run/kubernetes). If tlsCertFile and tlsPrivateKeyFile
+	// default /var/lib/kubelet/pki). If tlsCertFile and tlsPrivateKeyFile
```

Comment:
I agree with you that it probably just makes the `by default ...` part. A incorrect comment is worse than no comment haha :)

Reply:
Deleted this comment, PTAL.